### ISSUE TITLE
Pre-trained weights

### ISSUE THREAD

**Issue by LiGuo12** [Optimization-AI/LibAUC#66]
Hi all,

Is it possible to release your best weights? Our team will use them for research rather than commercial purposes, and will appropriately cite your work in the paper.

**Comment by optmai** [comment]
What task are you referring to?

**Comment by LiGuo12** [comment]
> What task are you referring to?

Hi, we are interested in the Deep AUC Maximization (DAM) model from the paper "Large-scale Robust Deep AUC Maximization: A New Surrogate Loss and Empirical Studies on Medical Image Classification". The model achieves 0.93 AUC on CheXpert.

**Comment by optmai** [comment]
Thank you for your interest! The result is actually an ensemble of several models trained using deep AUC maximization. You can follow the tutorial here https://github.com/Optimization-AI/LibAUC/blob/main/examples/05_Optimizing_AUROC_Loss_with_DenseNet121_on_CheXpert.ipynb to train a model. You can use different losses/optimizers available in our library to train different models and get an ensemble.  The original models for achieving 0.93 AUC on CheXpert were not available anymore.
